SUMMARY.md

```diff
@@ -63,6 +63,14 @@
     * [Search Aggregates](datasets/mozetl/search_aggregates/reference.md)
     * [Search Clients Daily](datasets/mozetl/search_clients_daily/reference.md)
 
+
+---
+
+* High-level Behavior
+    * Profile Behavior
+        * [Real World Usage](high_level/profile/realworldusage.md)
+        * [Profile History](high_level/profile/profilehistory.md)        
```

Comment:
For order, should we have the profile details (history, creation) first? Or the real-world usage?
Either way could make sense i guess.

Reply:
I think Real World Usage should come first, or be a level above the profile details. But I'm open to ideas!